lib/meilisearch/index.rb

```diff
@@ -220,6 +220,10 @@ def delete_documents!(documents_ids)
     alias delete_multiple_documents! delete_documents!
 
     def delete_document(document_id)
+      if document_id.nil? || document_id.to_s.empty?
+        raise MeiliSearch::Error::InvalidDocumentId, 'document_id cannot be empty or nil'
```

Comment:
Hi @Trehana this is a nice fix, but this class do not exist, so you must create it here: https://github.com/meilisearch/meilisearch-ruby/blob/main/lib/meilisearch/error.rb

It can just be inherited from the base `MeiliSearch::Error`